### PR TITLE
Initial Implementation.

### PR DESCRIPTION
**Pull request by taphu**
See commits for comments.

### CODE REVIEW

File: src/Network/Eureka/Cartographer/TH.hs [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell #-}
+{- |
+  Template haskell tools for compiling and incliding Elm files.
```
Comment (potomak) [inline src/Network/Eureka/Cartographer/TH.hs]: "including"

Comment (taphu) [inline src/Network/Eureka/Cartographer/TH.hs]: bah.

Comment (taphu) [inline src/Network/Eureka/Cartographer/TH.hs]: Also, this entire sentence is a copy and paste fubar. This package has nothing to do with Elm. :-(  I will fix this.

Comment (potomak) [inline src/Network/Eureka/Cartographer/TH.hs]: Oh, right!

**Pull request merged**